packages/broker/src/plugins/operator/MaintainOperatorValueService.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '@streamr/utils'
+import { MaintainOperatorValueHelper } from "./MaintainOperatorValueHelper"
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 * 24 // 1 day
+
+export class MaintainOperatorValueService {
+    config: OperatorServiceConfig
+    private checkValueInterval: NodeJS.Timeout | null = null
+    private helper: MaintainOperatorValueHelper
+
+    constructor(config: OperatorServiceConfig) {
+        logger.trace('MaintainOperatorValueService created')
+        this.config = config
+        this.helper = new MaintainOperatorValueHelper(config)
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    start(penaltyLimitFraction?: bigint): void {
+        logger.info('MaintainOperatorValueService started')
```

Comment:
The name of the module `MaintainOperatorValueService` will already be included in the log statement so you can simplify this to:
```suggestion
        logger.info('Started')
```